biz.aQute.bndlib/src/aQute/bnd/osgi/resource/ResourceUtils.java

```diff
@@ -198,9 +198,14 @@ public static ContentCapability getContentCapability(Resource resource) {
 
 	public static List<ContentCapability> getContentCapabilities(Resource resource) {
 		List<ContentCapability> result = new ArrayList<>();
-
-		for (Capability c : resource.getCapabilities(ContentNamespace.CONTENT_NAMESPACE)) {
-			result.add(as(c, ContentCapability.class));
+		if (resource == null) {
+			System.err.println("null resource");
+		}
+		List<Capability> capabilities = resource.getCapabilities(ContentNamespace.CONTENT_NAMESPACE);
+		if (capabilities != null) {
```

Comment:
This can never be null. The contract for the getCapabilities method says it cannot return null.